lib/ui/text_form_field.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart' as M;
```

Comment:
why u need it as M🤔

Reply:
Because there are conflicts. In the material package, there's already a Widget named TextFormField

Reply:
That is why you should rename your widget. This import remapping will need to be repeated for every page that uses the widget. You can rename it to 'CustomTextFormField' or something else.

Reply:
Fixed